chef/automate
# Enable Content with Credentials

### :nut_and_bolt: Description: What code changed, and why?
Prevent a user from viewing the content library until they have entered a correct token. A user can not list, install, or download content before applying the credentials. A new modal is displayed when visiting the CDS page. This modal allows the token to be applied. 

### :chains: Related Resources
#3902 

### :athletic_shoe: How to Build and Test the Change
1. `build components/automate-cds && build components/automate-gateway && build components/automate-ui-devproxy && start_all_services && start_automate_ui_background && ui_logs`
1. Go to https://a2-dev.test/cds
1. Ensure you see the image below. 
1. Enter "55c4f0d3-b982-4f39-93a7-01500966d291" for the Client ID and Secret ID.
1. Enter "https://chef-cloud.io" for the Tenant Specific URL
1. Click "Apply Credentials". 
1. Ensure the content items are displayed after. 

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
![submit_creds](https://user-images.githubusercontent.com/1679247/86947990-690fe980-c101-11ea-9942-56cc0555487c.gif)
